PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -4625,15 +4650,6 @@ intptr_t CALLBACK FindIncrementDlg::run_dlgProc(UINT message, WPARAM wParam, LPA
 			return NppDarkMode::onCtlColorDarker(reinterpret_cast<HDC>(wParam));
 		}
 
-		case WM_PRINTCLIENT:
-		{
-			if (NppDarkMode::isEnabled())
-			{
-				return TRUE;
-			}
-			break;
-		}
-

```

Comment:
Why? This will make buttons have white corners.
https://github.com/notepad-plus-plus/notepad-plus-plus/pull/11506

Reply:
Deletion of these lines makes regression.